src/bin.rs

```diff
@@ -127,6 +158,10 @@ impl Options {
                 .add_option(&["-d", "--dir"], Store, "destination dir");
             ap.refer(&mut add_newline)
                 .add_option(&["-n", "--add-newline"], StoreTrue, "add newline to the password");
+            ap.refer(&mut chunking)
+                .add_option(&["--chunking"], Store, "chunking algorithm (bup - default)");
+            ap.refer(&mut chunk_size)
+                .add_option(&["--chunk_size"], Store, "chunking size, default: 128k");
```

Comment:
`-` instead of `_` ? `--add-newline` already uses `-` .

I also wonder if maybe it should be `--chunk-algo` or something. No strong feelings about it though. :)